source/wp-content/themes/wporg-developer-2023/src/code-comment-form/block.php

```diff
@@ -53,10 +53,12 @@ function render( $attributes, $content, $block ) {
 		) . '</p>';
 	}
 
-	$title_block = sprintf(
-		'<!-- wp:heading {"fontSize":"heading-4"} --><h2 class="wp-block-heading has-heading-4-font-size">%s</h2><!-- /wp:heading -->',
+	$ordered_comments = wporg_developer_get_ordered_notes();
+
+	$title_block = empty( $ordered_comments ) ? sprintf(
+		'<!-- wp:heading --><h2 class="wp-block-heading">%s</h2><!-- /wp:heading -->',
 		__( 'User Contributed Notes', 'wporg' )
-	);
+	) : '';
```

Comment:
If there are comments the heading will be displayed at the top of that block, and won't be required again here.